2024-social-media-posts.md

```diff
@@ -0,0 +1,44 @@
+# IWD 2024 Social Media Campaign
+
+## Post 1
+Alt: A map of the world, with purple dots showing the coordinates of each R-Ladies Chapter.
+Image: plots/chapters_maps/2024-chapter-map.png
+Text:
+Today is International Women’s Day! 
+It’s an important day for us (although the R-Ladies mission is not limited to women).
+💜Who are we? R-Ladies is a worldwide organization whose mission is to promote gender diversity in the R community.
+💜Why do we exist? The R community suffers from an under-representation of women minority genders in every role and area of participation.
+For the rest of day, we’ll be highlighting ways that you can get involved with R-Ladies and be a better ally!
+#IWD2024 #RLadies 
+
+## Post 2
+Alt: Bar chart showing number of R-Ladies events per year.
+Image: plots/chapters_maps/2024-events-bar.png
+Text: 
+Where can you find your nearest R-Ladies Chapter?
+💜 R-Ladies has a global network of chapter organizing local and remote events thanks to the volunteer work of organizers! Find out more on Meetup: https://www.meetup.com/pro/rladies/
+💜 We also have R-Ladies Remote – an online, international chapter, especially for people that do not have local chapter and/or work remotely: https://www.meetup.com/rladies-remote/ 
+💜 You can find recordings of many events on our YouTube channel: https://www.youtube.com/channel/UCDgj5-mFohWZ5irWSFMFcng 
+#IWD2024 #RLadies
+
+## Post 3
+Alt: Bar chart showing number of R-Ladies Chapters having their first event per year.
+Image: plots/chapters_maps/2024-first-events.png
+Text:
+Would you like to start your own R-Ladies Chapter?
+💜 Read our guide new chapter organisers: https://guide.rladies.org/organization/intro/get-started/ 
+💜 We even have a mentoring program for new chapters! Read it here; https://guide.rladies.org/coordination/mentoring/ 
+💜 If you’re new to R-Ladies, another way to engage is through our community Slack channel: https://guide.rladies.org/comm/slack/ 
+💜 We already have two new chapters with their first events this year!
+#IWD2024 #RLadies
+
+## Post 4
+Image: R-Ladies logo (or IWD gif)
+Text:
+How can you help R-Ladies as an ally? 
+💜 Support your local chapter with a free venue and sponsorship
+💜 Invitations to speaks at conferences and highlighting job opportunities
+💜 Volunteer as a conference abstract reviewer: https://guide.rladies.org/comm/abstract-review/ 
+💜 Consult our R-Ladies directory: https://rladies.org/directory/ 
+💜 Donate to the global R-Ladies organization
```

Comment:
We should add a link: https://www.paypal.com/donate/?hosted_button_id=VB4EK67MJ373E

@drmowinckels should we create an rladies redirect to the paypal?